demos/imagetagger/pytest.ini

```diff
@@ -11,7 +11,10 @@ addopts =
 
     # show values of the local vars in errors:
     --showlocals
+asyncio_mode = auto
 filterwarnings =
     error
+    # Not sure what is causing this warning...
```

Comment:
Can't find it right now but I saw in the diff somewhere above, a BytesIO object being constructed inline as an argument to some PIL invocation. That's your unclosed file descriptor. It should be fixed by using a context manager there.

Reply:
Unfortunately, that's not the one. Still get the same warning.

Reply:
@Dreamsorcerer that is the one, you just applied the wrong fix. You're making sure to close the image object but not the underlying `BytesIO` object that is what causing the warnings in the first place.

Reply:
Do this: https://github.com/aio-libs/aiohttp-demos/pull/196/files/97969478c8ecded46ca45b3bf50c53e3b863ee7a#r1009870281.